lib/Doctrine/Common/Annotations/AnnotationReader.php

```diff
@@ -101,7 +101,9 @@ class AnnotationReader implements Reader
         // PlantUML
         'startuml' => true, 'enduml' => true,
         // Symfony 3.3 Cache Adapter
-        'experimental' => true
+        'experimental' => true,
+        // Slevomat Coding Standard
+        'phpcsSuppress' => true
```

Comment:
Can you please add a trailing comma here? As you can see, forgetting it is causing diff noise here